CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
 
+2.0.0-beta13 (in progress)
+==========================
+
+New Features
+------------
+
+- In a **Group Room**, you can now have more control over how your available bandwidth
+  is distributed among the RemoteVideoTracks that you have subscribed to. twilio-video.js
+  introduces the [Bandwidth Profile API](TODO(mmalavalli)). Note that this feature is
+  currently in **private beta** and hence will be **opt-in**. Please reach out to
+  [email@address.com](TODO(mmalavalli)) for more information about how to enable these
+  APIs for your Twilio Account. **Using these APIs in a Peer-to-Peer Room will have no effect**.
+  
+  ### Track Priority
+
+  While publishing a LocalTrack, you can now optionally specify its publish priority
+  in the following way:
+
+  ```js
+  const localTrackPublication = await room.localParticipant.publishTrack(localTrack, {
+    priority: 'high' // LocalTrack's publish priority - "low", "standard" or "high"
+  });
+
+  // LocalTrackPublication has a new property "priority" which stores the publish
+  // priority set while publishing the corresponding LocalTrack.
+  assert.equal(localTrackPublication.priority, 'high');
+  ```
+
+  This signals to the media server that this LocalTrack is more important than other
+  LocalTracks you may be publishing to the Room. The media server takes this into
+  account while allocating a subscribing RemoteParticipant's bandwidth to the LocalTrack.
+  If you do not specify a priority, then it defaults to `standard`.
+  
+  You can also find out about the priorities of RemoteTracks published by other
+  RemoteParticipants by accessing a new property `publishPriority` on the corresponding
+  RemoteTrackPublications:
+
+  ```js
+  remoteParticipant.on('trackPublished', remoteTrackPublication => {
+    console.log(`RemoteParticipant published a Track with priority ${remoteTrackPublication.publishPriority}`);
+  });
+  ```
+
+  ### Switching on/off RemoteVideoTracks
+
+  When a subscribing Participant's bandwidth is considerably low, the media server
+  tries to preserve higher priority RemoteVideoTracks by switching off lower priority
+  RemoteVideoTracks, which will stop receiving media until the media server decides
+  to switch them back on. You can now get notified about these actions by listening
+  to the `switchedOff` and `switchedOn` events on the RemoteVideoTrack:
+
+  ```js
+  remoteTrackPublication.on('subscribed', remoteTrack => {
+    remoteTrack.on('switchedOff', () => {
+      // You can now determine whether a particular RemoteTrack is switched off.
+      assert.equal(remoteTrack.isSwitchedOff, true);
+      console.log(`The RemoteTrack ${remoteTrack.name} was switched off`);
+    });
+
+    remoteTrack.on('switchedOn', () => {
+      assert.equal(remoteTrack.isSwitchedOff, false);
+      console.log(`The RemoteTrack ${remoteTrack.name} was switched on`);
+    });
+  });
+  ```
+
+  ### Bandwidth Profile
```

Comment:
I think make this the first item (before track prio). This is the "easy" API and is used during connect.